README.md

```
@@ -1,12 +1,12 @@
 <div align="center">
 
-<img src="docs/assets/arancel-mx-banner.svg" alt="arancel-mx - datos arancelarios de México reproducibles, auditables y trazables" width="100%" />
+<img src="docs/assets/arancel-mx-banner.svg" alt="arancel.mx - datos arancelarios abiertos de México" width="100%" />
 
 # arancel-mx
 
-### Datos arancelarios de México, reproducibles, auditables y trazables
+### De publicaciones oficiales dispersas a datos arancelarios verificables y listos para usar
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)